### PR TITLE
Add TU Munich

### PR DESCRIPTION
**Pull request by gernotheiser**
Signed-off-by: Gernot Heiser <gernot@unsw.edu.au>

### CODE REVIEW

**Review by lsf37 — APPROVED** [review]

File: content/Foundation/Membership/LOGOS/secondstate.svg [hunk 1]
```diff
@@ -1,71 +1,133 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="图层_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 595.28 595.28" style="enable-background:new 0 0 595.28 595.28;" xml:space="preserve">
-<style type="text/css">
+
+<svg
+   version="1.1"
+   id="图层_1"
```
Comment (axel-h) [inline content/Foundation/Membership/LOGOS/secondstate.svg:6]: Is everybody confident in using Unicode? ;)

File: content/news.pml [hunk 2]
```diff
@@ -39,6 +39,25 @@
...
+  13 July 2021: TU Munich joins seL4 Foundation
+</div>
+<div>
+ <p>
+   <img src="/Foundation/Membership/LOGOS/TU-Munich.svg"
+   style="width: 15%;  padding-left:10px;  float:right"
+	alt="TU Munich logo">
+
+   The seL4 Foundation is pleased to
+   welcome <a href="https://www.tum.de/en">ETH Zurich</a> as
```
Comment (axel-h) [inline content/news.pml:52]: uhm, I assume it should say `Technical University of Munich` (or 'TU München') here and not `ETH Zurich`?

**Review by axel-h — CHANGES_REQUESTED** [review]

File: content/news.pml (hunk 2 above)
Comment (gernotheiser) [reply content/news.pml]: Thanks for that catch, obviously a C&P error that would have been rather embarrassing.

File: content/Foundation/Membership/LOGOS/secondstate.svg (hunk 1 above)
Comment (gernotheiser) [reply content/Foundation/Membership/LOGOS/secondstate.svg]: Good point, I removed that

File: content/home.pml [hunk 3]
```diff
@@ -28,17 +28,17 @@
     <ul>
       <li>
         <strong>
-	  8 Jul 2021: Second State joins the seL4 Foundation
+	  13 Jul 2021: TU Munich joins the seL4 Foundation as an Associate Member
```
Comment (axel-h) [inline content/home.pml:31]: "Technical University of Munich (TUM) joins ..." would be more compliant with their style guide.

**Review by axel-h — APPROVED** [review]
Loog good now. The remaining comments are just minor.

File: content/Foundation/Membership/LOGOS/secondstate.svg [hunk 4]
```diff
@@ -1,71 +1,132 @@
...
+   showgrid="false"
+   inkscape:zoom="1.4665367"
+   inkscape:cx="297.29908"
+   inkscape:cy="297.64001"
+   inkscape:window-width="1803"
+   inkscape:window-height="1081"
+   inkscape:window-x="0"
+   inkscape:window-y="0"
+   inkscape:window-maximized="0"
+   inkscape:current-layer="图层_1" />
```
Comment (axel-h) [inline content/Foundation/Membership/LOGOS/secondstate.svg:42]: drop the chinese chars here?

Comment (lsf37) [reply content/Foundation/Membership/LOGOS/secondstate.svg:42]: I don't think they are a problem, svg/xml is perfectly fine with utf8 and it does specify utf8 as the encoding here. Chinese characters are no more special than any other ones.

**Pull request merged**